examples/with-one-login-many-subdomains/frontend/src/App.js

```diff
@@ -18,6 +18,11 @@ SuperTokens.init({
         websiteBasePath: "/",
```

Comment:
add email verification to this example app, and also the other example app (sub domain login one)